Added more context to the README

MinerMinerMods opened the pull request:
Discussed security and ML controllers providing additional context

MinerMinerMods committed:
Added more context to the README

Discussed security and ML controllers providing additional context

RandomHashTags commented on README.md:
```diff
@@ -67,11 +67,11 @@ Application and System Permissions will be a big part of this OS. This allows th
 ## AI/LLM
 
-Machine learning features are opt-in. We recognize they are powerful tools, but they should also be used when appropriate. They can be tightly woven into the OS and we will add security measures to prevent unauthorized behavior.
+Machine learning features are opt-in. We recognize they are powerful tools, but they should also be used when appropriate. They can be tightly woven into the OS and we will add security measures to prevent unauthorized behavior. Additionally, we want to add handles to applications to simplify models instead of having them find where to click on the windows and helps hide unintentional data from b ...
```
I think `handles` should be replaced with `AI accessibility`, and `simplify models` with `simplify and enhance model interaction and automation`; missing ending punctuation.

RandomHashTags commented on README.md:
```diff
@@ -67,11 +67,11 @@ Application and System Permissions will be a big part of this OS. This allows th
 ## Security
 
-The OS is mainly targeting the Swift Programming Language, which is a memory-safe language.
+The OS is mainly targeting the Swift Programming Language, which is a memory-safe language. The Permissions system helps to mitigate the interactions in the system when the programs are not functioning properly. 
```
I think `The Permissions system prevents processes (and programs) to access unauthorized resources, essentially sandboxing them to a user's preference.` would better express the intended behavior of the permissions system. Thoughts?

RandomHashTags reviewed (changes requested):
LGTM, two minor comments.

MinerMinerMods committed:
Update README.md

Added Revisions, Added a Technical Note

MinerMinerMods committed:
Matched change to the Table of Contents

Kept the old anchor but renamed `AI/LLM` to `Machine Learning Integrations`

MinerMinerMods committed:
Provided more context and fixed consistency error

Provided a note encouraging the development of `schwifty-permissions`
Fixed a consistency error where `Machine Learning Integrations` was referred to as `Machine Learning Integration` within the in doc ToC
